tests/plugins/widget/manual/keystroke.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: widget, bug, 4.14.1, 3998
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea
+
+### Scenario 1
+
+1. Open console in dev tools.
+
+1. Place cursor inside editor.
+
+1. Press CTRL and Enter key.
+
+#### Expected
+
+Source mode is opened and no error appears in dev console.
+
+#### Unexpected
+
+Error appears in dev console or source code editor is not opened.
+
+### Scenario 2
```

Comment:
If you have two scenarios, manual test should be split into two tests. In this case, however it looks like it could be the continuation of the first scenario (since you are in the source mode at the end of the first scenario and then you start from source mode) - but in this case it shouldn't be marked as second scenario just further steps for the first one.